### PR TITLE
Fixes #16276 - Add content counter to cv version

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-yum.html [hunk 1]
```diff
@@ -4,13 +4,30 @@
...
     </thead>
 
     <tbody>
       <tr bst-table-row ng-repeat="repository in table.rows">
         <td bst-table-cell>{{ repository.name }}</td>
         <td bst-table-cell>{{ repository.product.name }}</td>
+        <td bst-table-cell>
+          <div>
+            <a ui-sref="products.repository.manage-content.packages({productId: repository.product.id, repositoryId: repository.id})"
+               ng-show="repository.content_type == 'yum'"
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-yum.html]: Seems like this `ng-show` could just go on the table cell itself or a containing div?  Also, what happens when the content type isn't yum?

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-yum.html [hunk 2]
```diff
@@ -4,13 +4,26 @@
...
       </tr>
     </thead>
 
     <tbody>
       <tr bst-table-row ng-repeat="repository in table.rows">
         <td bst-table-cell>{{ repository.name }}</td>
         <td bst-table-cell>{{ repository.product.name }}</td>
+        <td bst-table-cell>
+          <div>
+            <a ui-sref="products.repository.manage-content.packages({productId: repository.product.id, repositoryId: repository.id})">
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-yum.html]: Sorry I missed this before but this needs to be translated via the translate directive.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-yum.html [hunk 3]
```diff
@@ -4,13 +4,26 @@
...
+              {{ repository.content_counts.rpm }} Packages
+            </a>
+          </div>
+          <div>
+            <a ui-sref="errata({repositoryId: repository.id})">
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-yum.html]: Same comment as above.

**Comment by chris1984** [comment]
@waldenraines can I get another review please?

**Pull request merged**